eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -27,8 +27,8 @@
 {% block ezxmltext_field %}
 {% spaceless %}
     {% set field_value = field.value.xml|xmltext_to_html5 %}
-    {{ block( 'simple_block_field' ) }}
 {% endspaceless %}
+{{ block( 'simple_block_field' ) }}
```

Comment:
@andrerom: the spaceless tags exist on all the field templates, the purpose I assume to be for cleanliness of the generated HTML.

The fix is to remove it on `simple_block_field` (wrapping the xml output) - the only alternative I can imagine would be to change the behavior (fix?) of twig's spaceless tag (see title/description)

`ezxmltext_field` also needs spaceless removed  because they are nesting, but yes - it should be possible to remove them altogether.